package.json

```diff
@@ -87,6 +88,7 @@
         "@testing-library/react": "^10.4.8",
         "@testing-library/user-event": "^12.1.1",
         "@types/d3-drag": "1",
+        "@types/d3-request": "^1.0.6",
```

Comment:
Maybe I missed it – is this used somewhere in the PR?

Reply:
the `text` function is imported at the tope of `src/data-fetcher/bam/bam-worker.ts`. Since it was a JS file previously, there was no type checking and the compiler didn't care that we didn't have `d3-request` available.